app/src/main/java/com/mobilecoronatracker/ui/countrieslist/CountriesListAdapter.kt

```diff
@@ -0,0 +1,63 @@
+package com.mobilecoronatracker.ui.countrieslist
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import coil.api.load
+import com.mobilecoronatracker.databinding.ItemCountryReportBinding
+
+class CountriesListAdapter :
+    RecyclerView.Adapter<CountriesListAdapter.ViewHolder>()
+{
+    var deltas = emptyList<CountriesListViewModelable.CountryReport>()
+    var followListener: CountryFollowListener? = null
+    var shareReportListener: ShareCountryReportListener? = null
+    var countryAnalysisRequestListener: ShowCountryAnalysisListener? = null
+
+    inner class ViewHolder(private val binding: ItemCountryReportBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun bind(item: CountriesListViewModelable.CountryReport) {
+            if (item.countryReport.flagPath.isNotEmpty()) {
+                binding.countryFlag.load(item.countryReport.flagPath)
+            }
+            binding.alwaysVisible.setOnClickListener {
```

Comment:
Maybe whole layout should be clickable to make hiding easier?

Reply:
On the other hand it is easier to accidentally collapse it. Are you sure it is better approach to make it clickable as a whole?

Reply:
As agreed on the other channel, let's keep only 'always visible' part clickable.